instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -222,7 +222,7 @@ class FastAPIInstrumentor(BaseInstrumentor):
 
     @staticmethod
     def instrument_app(
-        app: fastapi.FastAPI,
```

Comment:
Sphinx autodoc was unhappy with the app arg in the docstring. I tried explicitly typing it within the docstring, adding the mock import to sphinx's conf - nothing helped. Removing the typing here fixed it :man_shrugging: